server/views/referrals/new/courseParticipations/index.njk

```diff
@@ -52,23 +31,47 @@
             "data-testid": "success-banner"
           }
         }) }}
-      {% endif %}
+      {% else %}
+        {{ govukInsetText({
+          classes: "govuk-!-margin-top-0",
+          text: historyText,
+          attributes: {
+            "data-testid": "history-text"
+          }
+        }) }}
+
+        {% if summaryListsOptions.length %}
+          <p class="govuk-body" data-testid="pre-history-paragraph">Add another programme if you know that he started or completed a programme which is not listed below or skip this section of the referral if the history is not known.</p>
+        {% else %}
+          <p class="govuk-body" data-testid="no-history-paragraph">The programme team may use information about a person's Accredited Programme history to assess whether they are suitable.</p>
 
-      {% if summaryListsOptions.length %}
-        <p class="govuk-body" data-testid="pre-history-paragraph">The history below shows {{ person.name }}'s history of Accredited Programmes. Add another programme if you know that they started or completed a programme which is not listed below.</p>
+          <p class="govuk-body" data-testid="no-history-paragraph">You can continue by adding a programme history or skip this section of the referral if the history is not known.</p>
+        {% endif %}
+      {% endif %}
 
-        {% for summaryListOptions in summaryListsOptions %}
-          {{ govukSummaryList(summaryListOptions) }}
-        {% endfor %}
+      {% for summaryListOptions in summaryListsOptions %}
+        {{ govukSummaryList(summaryListOptions) }}
+      {% endfor %}
 
-        {{ actionForm(secondaryActionText="Add another") }}
-      {% else %}
-        <h2 class="govuk-heading-m" data-testid="no-history-heading">There is no Accredited Programme history for {{ person.name }}.</h2>
+      <form action="{{ action }}" method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+        <input type="hidden" name="hasReviewedProgrammeHistory" value="true"/>
 
-        <p class="govuk-body" data-testid="no-history-paragraph">The programme team may use information about a person's Accredited Programme history to assess whether they are suitable. You can add a programme to this history or continue with the referral if the {{ person.name }}'s history is unknown.</p>
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: "Add a programme",
```

Comment:
I thought we (still) wanted this to be "Add another" if there were existing course participations, but it looks like the designs might have changed a little since we met to discuss the changes, so we may as well keep it simple